### PR TITLE
Stop deleting runs on hard deletion

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

Following the recent work around the run execution. This PR stops deleting all dust app runs when we hard delete a workspace. Those runs will be deleted automatically after 30 days anyway (see https://github.com/dust-tt/tasks/issues/726).

### CODE REVIEW

**Comment by spolu** [comment]
I don’t see the runs diff is that expected?

**Comment by flvndvd** [comment]
> I don’t see the runs diff is that expected?

What do you mean, the function still exists and will be used by the cron. This PR just removes the call to the stored function that was attempting to delete all the runs.

File: core/src/stores/postgres.rs [hunk 1]
```diff
@@ -89,10 +89,6 @@ impl Store for PostgresStore {
         let mut c = pool.get().await?;
         let tx = c.transaction().await?;
 
-        // Block executions, Runs & Runs joins: we execute a SQL function
-        tx.execute("SELECT delete_project_runs($1)", &[&project_id])
```
Comment (PopDaph) [inline core/src/stores/postgres.rs:93]: Can we also remove the procedure `delete_project_runs`?

Comment (flvndvd) [reply core/src/stores/postgres.rs:93]: I kept it at first but happy to remove it.

**Review by spolu — APPROVED** [review]
My bad LGTM

File: core/src/stores/postgres.rs (hunk 1 above)
Comment (flvndvd) [reply core/src/stores/postgres.rs:93]: Done!

Comment (PopDaph) [reply core/src/stores/postgres.rs:93]: Getting rid of the dark magic! thank you 🙏🏻

**Review by PopDaph — APPROVED** [review]

**Pull request merged**